Gemfile

```diff
@@ -30,6 +30,7 @@ gem 'resque'
 gem 'resque-pool', '~> 0.7.0'
 gem 'resque-web', require: 'resque_web'
 gem 'rsolr', '>= 1.0'
+gem 'rubocop-rspec'
```

Comment:
Not a big deal but I would move 'rubocop-rspec' to the ":development" or the ":development, :test" group since I don't think we'd be running rubocop in the production environment.

Reply:
Thanks @coblej! I've moved it under the ":development, :test" group.